joschahenningsen/TUM-Live
# Check audio codec on uploaded videos

### Motivation and Context

closes #1336

### Description

Audio codec is now also checked during filavigate to a Livestreame upload and if it is not in aac format it will be transcoded.

### Steps for Testing


1. Log in as prof or admin
2. Create a lecture by choosing upload video option
3. Upload a video with an audio codec that is not aac
4. After the video is available to watch, locate the file and ffprobe it to see the audio codec is changed

### Screenshots
